README.md

```diff
@@ -168,3 +190,72 @@ targets:
     dependencies:
       - "barback"
 ```
+
+#### Defining `Builder`s in your package (similar to transformers)
```

Comment:
Do we want to add a note that while the config can accept this information, bazelify won't necessary act on it